src/gui/touchscreenlayout.h

```diff
@@ -0,0 +1,94 @@
+// Luanti
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2024 grorp, Gregor Parzefall <grorp@posteo.de>
+
+#pragma once
+
+#include "client/texturesource.h"
+#include "irr_ptr.h"
+#include "irrlichttypes_bloated.h"
+#include "ITexture.h"
+#include "IGUIStaticText.h"
+#include <unordered_map>
```

Comment:
`ITexture`, `ISimpleTextureSource` and `IGUIStaticText` are optimally forwards-declared to minimize the size of this header.